packages/myst-cli/src/build/tex/single.ts

```diff
@@ -133,19 +153,25 @@ export async function localArticleToTexRaw(
   if (results.length === 1) {
     writeFileToFolder(output, results[0].value);
   } else {
+    let includeContent = '';
+    let fileInd = 0;
     const { dir, name, ext } = path.parse(output);
-    const includeFileBases = results.map((result, ind) => {
-      const base = `${name}-${content[ind]?.slug ?? ind}${ext}`;
-      const includeFile = path.format({ dir, ext, base });
-      let part = '';
-      const { title, content_includes_title } = content[ind]?.frontmatter ?? {};
-      if (title && !content_includes_title) {
-        part = `\\section{${title}}\n\n`;
+    articles.forEach((article) => {
```

Comment:
There is some nasty repetition among multipage exports for `raw tex`, `templated tex`, `raw typst`, `templated typst` that needs refactoring. I suspect this will continue to get worse as we start updating individual article frontmatter, etc. - eventually it will reach a breaking point. Not this PR though... 😅